tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/deid/remove_fitbit_data_if_max_age_exceeded_test.py

```diff
@@ -1,11 +1,11 @@
 """
```

Comment:
Wrong PR?

Reply:
that was because of changes in other PR, it is fixed now.